ext/standard/basic_functions.stub.php

```diff
@@ -596,3 +596,231 @@ function substr_compare(string $main_str, string $str, int $offset, int $length
+function php_ini_scanned_files() {}
+
+/** @return string|false */
+function php_ini_loaded_file() {}
+
+/* iptc.c */
+
+/** @return string|bool */
+function iptcembed(string $iptcdata, string $jpeg_file_name, int $spool = 0) {}
+
+/** @return array|false */
+function iptcparse(string $iptcblock) {}
+
+/* levenshtein.c */
+
+function levenshtein(string $str1, string $str2, $cost_ins = UNKNOWN, int $cost_rep = UNKNOWN, int $cost_del = UNKNOWN): int {}
+
+/* link.c */
+
+#if defined(HAVE_SYMLINK) || defined(PHP_WIN32)
+/** @return string|false */
+function readlink(string $path) {}
+
+/** @return int|false */
+function linkinfo(string $path) {}
+
+function symlink(string $target, string $link): bool {}
+
+function link(string $target, string $link): bool {}
+#endif
+
+/* mail.c */
+
+function ezmlm_hash(string $str): int {}
+
+/** @param string|array $additional_headers */
+function mail(string $to, string $subject, string $message, $additional_headers = UNKNOWN, $additional_parameters = ""): bool {}
+
+/* math.c */
+
+/**
+ * @param int|float $number
+ * @return int|float
+ */
+function abs($number) {}
+
+function ceil(float $number): float {}
+
+function floor(float $number): float {}
+
+/** @return float|false */
+function round($number, int $precision = 0, int $mode = PHP_ROUND_HALF_UP) {}
+
+function sin(float $number): float {}
+
+function cos(float $number): float {}
+
+function tan(float $number): float {}
+
+function asin(float $number): float {}
+
+function acos(float $number): float {}
+
+function atan(float $number): float {}
+
+function atanh(float $number): float {}
+
+function atan2(float $y, float $x): float {}
+
+function sinh(float $number): float {}
+
+function cosh(float $number): float {}
+
+function tanh(float $number): float {}
+
+function asinh(float $number): float {}
+
+function acosh(float $number): float {}
+
+function expm1(float $number): float {}
+
+function log1p(float $number): float {}
+
+function pi(): float {}
+
+function is_finite(float $number): bool {}
+
+function is_nan(float $number): bool {}
+
+function intdiv(int $dividend, int $divisor): int {}
+
+function is_infinite(float $number): bool {}
+
+/** @return int|float */
+function pow($base, $exp) {}
+
+function exp(float $number): float {}
+
+/** @return float|false */
+function log(float $number, float $base = UNKNOWN) {}
+
+function log10(float $number): float {}
+
+function sqrt(float $number): float {}
+
+function hypot(float $x, float $y): float {}
+
+function deg2rad(float $number): float {}
+
+function rad2deg(float $number): float {}
+
+/** @return int|float */
+function bindec(string $binary_string) {}
+
+/** @return int|float */
+function hexdec(string $hex_string) {}
+
+/** @return int|float */
+function octdec(string $octal_string) {}
+
+function decbin($number): string {}
+
+function decoct($number): string {}
+
+function dechex($number): string {}
+
+/** @return string|false */
+function base_convert($number, int $frombase, int $tobase) {}
```

Comment:
Tweaked in https://github.com/php/php-src/commit/c43179fb6bf31f1b27751c648e8cff5e8754803b, this can now be just `: string`.